catalog/catalog.yaml

```diff
@@ -258,6 +268,9 @@ components:
         apiKeyOpts:
           description: Configuration for API key. Must be provided if authType is api_key.
           $ref: '#/components/schemas/ApiKeyOpts'
+        BasicAuthOpts:
```

Comment:
This should be `basicOpts` to be consistent, since the type is called `Basic`. And the convention for the other opts has been TypeOpts (i.e. it's ApiKeyOpts, not ApiKeyAuthOpts). Also it should be lower case "b". The actual schema can be named `BasicAuthOpts` if you want, since that's not exposed in the API.

Reply:
oops that was a typo anyways, probably copy-pasted. Fixed.